### ISSUE TITLE
Food recipes return an extra bucket/bottle when made in Create's mechanical mixer

### ISSUE THREAD

**Issue by DerekSSWong** [nexnux/Satisfactoria-mod-pack#10]
Recipes such as this one:

![Alt text](https://i.imgur.com/VwmDUBY.png)

Applies for most if not all similar recipes, tested on food from Ars, Croptopia, and Supplementaries.

**Comment by nexnux** [comment]
An extra, as in it double the return of the iron bucket after use ?

**Comment by DerekSSWong** [comment]
Yes, one bucket of milk goes in, 2 empty buckets come out. Same goes for glass bottles
